odml/addons.py

```
@@ -0,0 +1,23 @@
+import odml.tools.xmlparser as X
+import odml.tools.jsonparser as Y 
+
+def load(filename, backend="xml"):
+	if backend == "xml":
+		return X.load(filename)
+	#elif backend == "json":
+	#	return Y.JSONReader().fromFile(open(filename))
+		
+
+def save(obj, path, backend="xml"):
+	if backend == "xml":
+		X.XMLWriter(obj).write_file(path)
+	elif backend == "json":
+		Y.JSONWriter(obj).write_file(path)
+
+def display(obj, backend = "xml"):
+	if backend == "xml":
+		print (str(X.XMLWriter(obj)))
+	elif backend =="json":
+		print (str(Y.JSONWriter(obj)))
+
```

Same as above (failing silently).